scripts/bench.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+import argparse
+import os
+import sys
+
+from monotonic import monotonic
+
+from detect_secrets.core.usage import PluginOptions
+
+if os.name == 'posix' and sys.version_info[0] < 3:
+    import subprocess32 as subprocess
+else:
+    import subprocess
+
+
+def getArguments():
```

Comment:
Please keep the style consistent for function naming (`snake_case` rather than `camelCase`).

Reply:
Fixed!